integration_tests/test_pv_connection.py

```diff
@@ -25,26 +25,26 @@ def on_market_cycle(self, market_info):
                 offer_info = json.loads(offer["offer"])
                 assert offer_info["price"] == 50 * market_info["available_energy_kWh"]
                 assert offer_info["energy"] == market_info["available_energy_kWh"]
-                # Validate that the bid was placed to the market
+                # Validate that the offer was placed to the market
                 offer_listing = self.list_offers()
-                listed_offer = next(bid for bid in offer_listing["offer_list"] if bid["id"] == offer_info["id"])
+                listed_offer = next(offer for offer in offer_listing["offer_list"] if offer["id"] == offer_info["id"])
                 assert listed_offer["price"] == listed_offer["price"]
                 assert listed_offer["energy"] == listed_offer["energy"]
-                # Try to delete the bid
+                # Try to delete the offer
                 delete_resp = self.delete_offer(offer_info["id"])
                 assert delete_resp["deleted_offer"] == offer_info["id"]
-                # Validate that the bid was deleted from the market
-                empty_listing = self.list_bids()
+                # Validate that the offer was deleted from the market
+                empty_listing = self.list_offers()
```

Comment:
What? We didnt see this in multiple PRs? Thanks for fixing!

Reply:
I really have no clue